Documentation/project-docs/api-review-process.md

```diff
@@ -1,51 +1,48 @@
-API Review Process
-==================
+# API Review Process
 
-The .NET Framework has a long standing history of taking API usability extremely seriously. Thus, we generally review every single API that is added to the product. This page discusses how we conduct API reviews for components that are open sourced.
+The .NET Framework has a long standing history of taking API usability extremely seriously. Thus, we generally review every single API that is added to the product. This page discusses how we conduct design reviews for components that are open sourced.
 
-## Process Goals
+## Which APIs should be reviewed?
 
-The key goals are:
+The rule of thumb is that we (**corefx**) review every API that is being added to the `System` namespace. In some cases, we also review APIs that are added to namespaces, such as `Microsoft`. We mostly do this for high impact APIs, such as Roslyn, and when both the owner of the technology and us feel there is win-win for both sides if we review the APIs. However, we can't scale to review all APIs being added to .NET.
```

Comment:
Probably a stupid question, but this means `System.*` and not only `System` right? The last sentence is a bit ambiguous about what is considered ".NET".